### PR TITLE
Remove hard coded path to file

### PR DESCRIPTION
**Pull request by bishboria**
- Fix tests
- Update Google credentials with proper key

### CODE REVIEW

File: config/google_drive_config.json [hunk 1]
```diff
@@ -5,5 +5,5 @@
     "https://www.googleapis.com/auth/drive",
     "https://spreadsheets.google.com/feeds/"
   ],
-  "refresh_token": "1/AUNh1aOdTESQBa4f4xqE5qVHNGnO-_2dEi4hF1woPYdIgOrJDtdun6zK6XiATCKT"
-}
+  "refresh_token": "1/NMmpVCcuqR5f1kaHxqKKCOqNJeeqzvzjhIvKW8ep-0I"
+}
```
Comment (benlovell) [inline config/google_drive_config.json:9]: Whoops.

**Comment by bradwright** [comment]
I know this is a relatively throwaway repo, but in the interests of good commit hygiene I would have expected this to be two separate commits (one to fix the creds, one to fix the tests).

File: config/google_drive_config.json (hunk 1 above)
Comment (bishboria) [inline config/google_drive_config.json:9]: This isn't a security issue, it just has to be in a file for Google API shenanigans

Comment (benlovell) [inline config/google_drive_config.json:9]: Sorry, I should have made myself clear - I was referring to the lack of terminating newline.

**Comment by bishboria** [comment]
Not forgotten about these comments, I'll get them sorted next.

**Pull request merged**